packages/app-project/webpack.config.js

```diff
@@ -1,13 +1,31 @@
 const path = require('path')
+const webpack = require('webpack')
 
 module.exports = {
+  plugins: [
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
+    }),
+  ],
   resolve: {
     alias: {
       '@components': path.resolve(__dirname, 'src/components'),
       '@helpers': path.resolve(__dirname, 'src/helpers'),
       '@screens': path.resolve(__dirname, 'src/screens'),
       '@shared': path.resolve(__dirname, 'src/shared'),
-      '@stores': path.resolve(__dirname, 'stores')
+      '@stores': path.resolve(__dirname, 'stores'),
+      process: 'process/browser'
+    },
+    fallback: {
+      console: false,
+      domain: false,
+      fs: false,
+      http: false,
+      https: false,
+      net: false,
+      os: false,
+      process: 'process/browser',
+      tls: false
```

Comment:
I haven't tested this with the actual app. The next.js webpack config loads up the aliasing portion of this, so I don't think this will conflict, but we need to confirm.

Reply:
I think the only package that uses these might be `@sentry/node`, and that shouldn't be included in webpack builds anyway.

The NextJS build doesn't use these, as far as I can tell, and runs fine in webpack 5 (unless Next is adding fallbacks itself in the core webpack config.)

Reply:
Having said that, I think it's going to be confusing to have a webpack config file that's only partly loaded into `next.config.js`. I'd assume that the entire config here is being applied to the NextJS build. If fallbacks are only needed for the storybook build, then I'd add them to the custom storybook config.

If they don't make any difference at all to either build, then I'd leave them out.